content/gsod/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Google Season of Docs with PyBaMM
```

Comment:
Could we add a summary here? It will show up on Google and other search engines below the link to the page.

```suggestion
title: Google Season of Docs with PyBaMM
summary: This page describes the process, idea lists, and other logistics for participation in the Google Season of Docs program with PyBaMM.
```